README.md

```diff
@@ -4,7 +4,7 @@ Welcome to your shiny new Codespace running Next.js! We've got everything fired
 
 You've got a blank canvas to work on from a git perspective as well. There's a single initial commit with the what you're seeing right now - where you go from here is up to you!
 
-Everything you do here is contained within this one codespace. There is no repository on GitHub yet. If and when you’re ready you can click "Publish Branch" and we’ll create your repository and push up your project. If you were just exploring then and have no further need for this code then you can simply delete your codespace and it's gone forever.
+Everything you do here is contained within this one Codespace. There is no repository on GitHub yet. If and when you’re ready you can click "Publish Branch" and we’ll create your repository and push up your project. If you were just exploring then and have no further need for this code then you can simply delete your Codespace and it's gone forever.
```

Comment:
Marketing guidelines call for using lowercase "codespace" when referring to an instance of a Codespace (the product) so this was correct before and needs to be reverted back. 🙇

Reply:
Unless I'm missing something this still looks changed when it should not be. Thanks!